framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -1255,6 +1261,15 @@ protected void addComputedTypeAnnotations(Element elt, AnnotatedTypeMirror type)
     }
 
     /**
+     * Adds default annotations to {@code type}.
```

Comment:
The class documentation states that `This implementation only adds qualifiers explicitly specified by the programmer.` and that `addComputedTypeAnnotations` should be overridden to customize how annotations are added.
Do we need a one-argument version of `addComputedTypeAnnotations` and use that name instead of the new `addDefaultAnnotations`? Or can we provide a `Tree` or an `Element` to use the already existing implementations in the call sites of `addDefaultAnnotations`?
The javadoc should at least discuss what the relationship to the existing methods is.
I would add a big TODO to this method that this API needs to be cleaned up.